packages/e2e-tests/puppeteer/scripts/rps.ts

```diff
@@ -2,76 +2,76 @@ import {setUpBrowser, loadRPSApp, waitForAndClickButton} from '../helpers';
 import {Page} from 'puppeteer';
 
 export async function setupRPS(rpsTabA: Page, rpsTabB: Page): Promise<void> {
-  async function tabA(): Promise<void> {
-    await waitForAndClickButton(rpsTabA, 'Start Playing!');
-    await (await rpsTabA.waitFor('#name')).type('playerA');
-    await waitForAndClickButton(rpsTabA, 'Connect with MetaMask');
+  async function playerA(page: Page): Promise<void> {
+    await waitForAndClickButton(page, 'Start Playing!');
+    await (await page.waitFor('#name')).type('playerA');
+    await waitForAndClickButton(page, 'Connect with MetaMask');
   }
-  async function tabB(): Promise<void> {
-    await waitForAndClickButton(rpsTabB, 'Start Playing!');
-    await (await rpsTabB.waitFor('#name')).type('playerB');
-    await waitForAndClickButton(rpsTabB, 'Connect with MetaMask');
+  async function playerB(page: Page): Promise<void> {
+    await waitForAndClickButton(page, 'Start Playing!');
+    await (await page.waitFor('#name')).type('playerB');
+    await waitForAndClickButton(page, 'Connect with MetaMask');
   }
 
-  await Promise.all([tabA(), tabB()]);
+  await Promise.all([playerA(rpsTabA), playerB(rpsTabB)]);
 }
 
 export async function clickThroughRPSUI(rpsTabA: Page, rpsTabB: Page): Promise<void> {
-  async function tabA(): Promise<void> {
-    const walletIFrameA = rpsTabA.frames()[1];
+  async function playerB(page: Page): Promise<void> {
+    const walletIFrame = page.frames()[1];
     // NOTE: There is some weird scrolling issue. .click() scrolls and somehow React re-renders this
     // button and so we get a "Node is detached from document error". Using .evaluate() fixes it.
     // https://github.com/puppeteer/puppeteer/issues/3496
-    await (await rpsTabA.waitForXPath('//button[contains(., "Create a game")]')).evaluate(
+    await (await page.waitForXPath('//button[contains(., "Create a game")]')).evaluate(
```

Comment:
This is what [*defines*](https://github.com/statechannels/monorepo/blob/12f65a4cd4083a2a0a8171f284d8ca1e9fb53362/packages/rps/notes/user-workflows-new-api.png) `playerB`, and was previously attached to `tabA`.